tests/sanity/ignore.txt

```diff
@@ -550,3 +550,9 @@ test/units/utils/test_shlex.py future-import-boilerplate
 test/units/utils/test_shlex.py metaclass-boilerplate
```

Comment:
Please rename the file as ignore-2.10.txt for ansible-test to ignore these lines with ansible-2.10
Similar file should be maitained for ansible-2.9 as well as certification test will happen on ansible-2.9, but as Felix said you should also avoid ignore file and try to fix the error like I did in the last pull request unless absolutely necessary

Reply:
I have made the above modifications - renaming and added ignore statements but still I see 306 errors have not reduced.